portfolio/src/main/webapp/BusinessInfoForm.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <h1>Business Form</h1>
+    <p>In this project, we want to give small businesses a platform and an opportunity to make themselves
+        more accessible to nearby customers. Please enter the information down below to be featured on our map.
+    </p>
+    <form action="/form-handler" method="POST">
```

Comment:
Do you also have the servlet that handles this request?

Reply:
Hi Tali, I am currently working on it. I am planning to make that a separate pull request tomorrow :)

Reply:
Hi Tali! Just letting you know I have added the servlet to go along with the business form post request

Reply:
Can you add the servlet to this PR? I think it would be a little weird to submit to main HTML that calls an endpoint that doesn't exist yet :)

Reply:
Oh whoops I see it now